src/flash/core/integrations/pytorch_tabular/backbones.py

```diff
@@ -76,6 +76,7 @@ def load_pytorch_tabular(
             OmegaConf.create(parameters),
             OmegaConf.to_container(model_config),
         )
+        config.embedded_cat_dim = 0  # todo: not sure if this is right
```

Comment:
That is strange as `config` includes `embedding_dims`
and checking the tabular codebase it goes to
```py
        if hasattr(self.hparams, "_backbone_input_dim"):
            _curr_units = self.hparams._backbone_input_dim  # TODO implement this backdoor in every model?
        else:
            _curr_units = self.hparams.embedded_cat_dim + self.hparams.continuous_dim
```

Reply:
Oh yeah, I forgot to mention that. So, once you put in all the different configs into TabularModel, they are all merged together into a single config in the init. Then, during fit (we get access to the data), we infer other configs from data into InferredConfig. And then, when the PyTorch Lightning Model is initialized, we combine the config, and inferred_config into a single config. 

I know, it's a little counter-intuitive, but was merged so that while coding new models, you can access everything from a single config object without having to remember the lineage of each config.

Reply:
Well, I ll skip these embedding cases for now, but if you could have a look at it would be great!